### PR TITLE
Support to load orc compression file

### PR DESCRIPTION
**Pull request by Linary**

### CODE REVIEW

File: src/main/java/com/baidu/hugegraph/loader/reader/LineFetcher.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+package com.baidu.hugegraph.loader.reader;
+
+import java.io.IOException;
+import java.util.List;
+
+import com.baidu.hugegraph.loader.source.InputSource;
+import com.baidu.hugegraph.util.E;
+
+/**
+ * Maybe can as base class for JDBC RowFetcher
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/loader/reader/LineFetcher.java]: can be be used as

File: src/main/java/com/baidu/hugegraph/loader/reader/file/FileLineFetcher.java [hunk 2]
```diff
@@ -26,227 +26,211 @@
...
-                    LOG.warn("Failed to close stream of '{}'", readable);
-                }
-            }
-            throw new LoadException("Failed to create reader for '%s'",
-                                    e, readable);
+    /**
+     * Just match header for second or subsequent file first line
+     */
+    private boolean checkMatchHeader(String line) {
+        if (!this.source().format().needHeader() || !this.fresh) {
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/loader/reader/file/FileLineFetcher.java]: prefer rename fresh to firstLine

File: src/main/java/com/baidu/hugegraph/loader/reader/file/FileLineFetcher.java [hunk 3]
```diff
@@ -26,227 +26,211 @@
...
+            return false;
         }
+        this.fresh(false);
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/loader/reader/file/FileLineFetcher.java]: don't update here, prefer do it in increaseOffset()

File: src/test/resources/orc_compress_file/schema.groovy [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+schema.propertyKey("p_boolean").asBoolean().ifNotExist().create();
+schema.propertyKey("p_byte").asByte().ifNotExist().create();
+schema.propertyKey("p_int").asInt().ifNotExist().create();
+schema.propertyKey("p_long").asLong().ifNotExist().create();
+schema.propertyKey("p_float").asFloat().ifNotExist().create();
+schema.propertyKey("p_double").asDouble().ifNotExist().create();
+schema.propertyKey("p_string").asText().ifNotExist().create();
+schema.propertyKey("p_date").asDate().ifNotExist().create();
+
+schema.vertexLabel("person").properties("name", "p_boolean", "p_byte", "p_int", "p_long",
```
Comment (zhoney) [inline src/test/resources/orc_compress_file/schema.groovy]: too ugly align
```
schema.vertexLabel("person")
      .properties("name", "p_boolean", "p_byte", "p_int", "p_long", "p_float", "p_double", "p_string", "p_date")
      .primaryKeys("name")
      .ifNotExist().create();
```

**Pull request merged**